extensions/amp-form/0.1/test/test-form-proxy.js

```diff
@@ -33,6 +35,22 @@ describes.repeated('installFormProxy', {
 }, (name, variant) => {
   let form;
   let inputs;
+  let sandbox;
+
+  before(() => {
+    sandbox = sinon.sandbox.create();
```

Comment:
there shouldn't be a need to create a `sandbox` the new `describes` provides a global, already-instantiated `sandbox`

Reply:
I get `ReferenceError` if I try to access `sandbox` globally. I don't  think `repeated` sets up the environment.

Reply:
this is correct